src/reducers/splitSentences.ts

```diff
@@ -0,0 +1,39 @@
+import _ from 'lodash'
+import { State } from '../util/initialState'
+import { contextOf, head, headRank, headValue, pathToContext, reducerFlow } from '../util'
+import { existingThoughtChange, newThought, setCursor } from '../reducers'
+import { ROOT_TOKEN } from '../constants'
+
+/** Split thought by sentences. Create new thought for each sentence. Thought value, on which cursor is on, replace with first sentence. */
+const splitSentences = (state: State) => {
+  const { cursor } = state
+  if (!cursor) return state
+  const thoughts = pathToContext(cursor)
+  const cursorContext = thoughts.length > 1 ? contextOf(thoughts) : [ROOT_TOKEN]
```

Comment:
You can also use [rootedContextOf](https://github.com/cybersemics/em/blob/dev/src/util/rootedContextOf.ts).